.github/workflows/tagging.yml

```
@@ -0,0 +1,26 @@
+name: Create a periodical tag
+
+on:
+  schedule:
+    - cron:  '0 0 * * 0'
+
+env:
+  # Account for committing
+  USER_NAME: "Universal-ctags CI"
+  USER_EMAIL: "Universal-ctags-ci@users.noreply.github.com"
```

```suggestion
  USER_EMAIL: "Universal-ctags@users.noreply.github.com"
```
GitHub provides email addresses with the `<username>@users.noreply.github.com` format for uses who don't want to open their actual email addresses.
I'm not sure `<organizationname>@users.noreply.github.com` is a valid address, but, at least, the `-ci` should be removed.